hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -215,7 +216,12 @@ private void moveServerRegionsFromGroup(Set<Address> servers, String targetGroup
           if (!targetGrp.containsTable(region.getTable())) {
             LOG.info("Moving server region {}, which do not belong to RSGroup {}",
                 region.getShortNameToLog(), targetGroupName);
-            this.master.getAssignmentManager().move(region);
+            try {
+              this.master.getAssignmentManager().move(region);
+            }catch (IOException ioe){
+              LOG.error("Move region {} from group failed, will retry, current retry time is {}",
```

Comment:
May get too verbose. How about log as debug on each retry, then if max number of reties has been reached, log as error?

Reply:
Yes, I agree. But I think we need failed regions' names to move them to the target group servers after the failed call, because we can't call move tables or servers methods to move failed regions again.Maybe I can add a failed regions list in the ex message when max number of reties has been reached.What do you think about it?

Reply:
I have changed this log level to DEBUG, and added a failed regions list in the ex message when max number of reties has been reached.